crop_health_api/custom_openapi.py

```diff
@@ -17,42 +17,193 @@ def custom_openapi_gen(openapi_schema: dict, example_code_dir: Path):
+                                        "type": "string",
+                                        "description": "Error message.",
+                                    },
+                                },
+                            }
+                        }
+                    },
+                },
+                "500": {
+                    "description": "Internal Server Error",
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "required": ["code", "type", "message"],
+                                "properties": {
+                                    "code": {
+                                        "type": "integer",
+                                        "description": "Error code.",
+                                    },
+                                    "type": {
+                                        "type": "string",
+                                        "description": "Error type.",
+                                    },
+                                    "message": {
+                                        "type": "string",
+                                        "description": "Error message.",
+                                    },
+                                },
+                            }
+                        }
+                    },
+                },
+                "503": {
+                    "description": "No worker is available to serve request",
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "required": ["code", "type", "message"],
+                                "properties": {
+                                    "code": {
+                                        "type": "integer",
+                                        "description": "Error code.",
+                                    },
+                                    "type": {
+                                        "type": "string",
+                                        "description": "Error type.",
+                                    },
+                                    "message": {
+                                        "type": "string",
+                                        "description": "Error message.",
+                                    },
+                                },
+                            }
+                        }
+                    },
+                },
+            }
+
+            # Successful reponse
+            openapi_schema["paths"][endpoint_path][method]["responses"]["200"] = {
+                "description": (
+                    "Predicted class confidences, all summing to 1.0. Actual class names and number of returned classes may vary.",
```

Comment:
Given that we now have separate endpoints for each model, the text about class names varying can be removed.